SPECS/qemu.spec

```
@@ -232,6 +232,15 @@ cp -r scripts/qmp %{buildroot}%{_datarootdir}/qemu
 %{?_cov_results_package}
 
 %changelog
+* Tue Oct 21 2025 Tu Dinh <ngoc-tu.dinh@vates.tech> - 4.2.1-5.2.14.1
```

Nitpicking: Update the date to reflect a more accurate merge date
```suggestion
* Tue Nov 04 2025 Tu Dinh <ngoc-tu.dinh@vates.tech> - 4.2.1-5.2.14.1
```

---

Done. How do we choose the changelog date? I thought it should be set to the last change date of the merge PR.

---

From my understanding, it's the date of our PR. The closer to the release date the better.

---

We're not too strict on that, but when there's more than a few days between the PR and the merge, it can be good to update the date indeed.